2025-03-15 13:15:39,868 [INFO] USER:racharla_sanjana s3_resource.py:40:validate_bucket_name LOG: Starting validating bucket name 'syndicate-education-platform-custom-sandbox-artifacts-6675'
2025-03-15 13:15:39,868 [INFO] USER:racharla_sanjana s3_resource.py:73:validate_bucket_name LOG: Finished validating bucket name 'syndicate-education-platform-custom-sandbox-artifacts-6675'
2025-03-15 13:15:39,868 [INFO] USER:racharla_sanjana helper.py:525:convert LOG: Stripping run_id:JAP-11,run_type:stm,type:user,user_id:f452cf26 from "," a bit..
2025-03-15 13:15:39,868 [INFO] USER:racharla_sanjana helper.py:529:convert LOG: Converting: run_id:JAP-11,run_type:stm,type:user,user_id:f452cf26 to dict..
2025-03-15 13:15:39,869 [INFO] USER:racharla_sanjana helper.py:541:convert LOG: Converted to such a dict: {'run_id': 'JAP-11', 'run_type': 'stm', 'type': 'user', 'user_id': 'f452cf26'}
2025-03-15 13:15:41,187 [WARNING] USER:racharla_sanjana generator.py:76:generate_configuration_files LOG: The config_path property is not specified. The working directory is used
2025-03-15 13:15:41,187 [ERROR] USER:racharla_sanjana __init__.py:44:_mkdir LOG: [WinError 183] Cannot create a file when that file already exists: 'C:\\Users\\racharla_sanjana\\Documents\\Aws\\AWS\\task04\\.syndicate-config-dev'
2025-03-15 13:15:41,187 [WARNING] USER:racharla_sanjana generator.py:93:generate_configuration_files LOG: The "project_path" property is not specified. The working directory will be used as a project path. To change the path, edit the syndicate.yml by path C:\Users\racharla_sanjana\Documents\Aws\AWS\task04\.syndicate-config-dev
2025-03-15 13:15:41,190 [INFO] USER:racharla_sanjana generator.py:155:generate_configuration_files LOG: Syndicate initialization has been completed. 
Set SDCT_CONF:
Unix: export SDCT_CONF="C:\Users\racharla_sanjana\Documents\Aws\AWS\task04\.syndicate-config-dev"
Windows: setx SDCT_CONF C:\Users\racharla_sanjana\Documents\Aws\AWS\task04\.syndicate-config-dev
2025-03-15 13:15:41,191 [INFO] USER:racharla_sanjana helper.py:330:timed LOG: Stage config, elapsed time: 0:00:01.321945
2025-03-15 13:15:59,274 [INFO] USER:racharla_sanjana handlers.py:106:syndicate LOG: Configuration used: C:\Users\racharla_sanjana\Documents\Aws\AWS\.syndicate-config-dev
2025-03-15 13:15:59,446 [INFO] USER:racharla_sanjana processor.py:143:_prepare_bucket_view LOG: Viewing complement, URIBucketView, has been found, setting up the raw data.
2025-03-15 13:16:00,970 [INFO] USER:racharla_sanjana project_state.py:122:get_remote LOG: Unsafely loaded project state file from S3 bucket. The retrieved object has type: dict
2025-03-15 13:16:00,970 [INFO] USER:racharla_sanjana project_state.py:127:get_remote LOG: Made ProjectState object from the dict loaded from S3 bucket
2025-03-15 13:16:01,694 [INFO] USER:racharla_sanjana sync_processor.py:53:sync_project_state LOG: Project state file has been successfully synced
2025-03-15 13:16:01,694 [INFO] USER:racharla_sanjana helper.py:647:resolve_project_path LOG: Parameter: 'project_path' wasn't specified. Getting automatically
2025-03-15 13:16:01,694 [INFO] USER:racharla_sanjana helper.py:651:resolve_project_path LOG: Path: 'C:/Users/racharla_sanjana/Documents/Aws/AWS' was assigned to the parameter: 'project_path'
2025-03-15 13:16:01,695 [INFO] USER:racharla_sanjana helper.py:503:convert LOG: Checking whether eu-central-1 is a valid region...
2025-03-15 13:16:01,695 [INFO] USER:racharla_sanjana helper.py:508:convert LOG: Value 'eu-central-1' is a valid region, returning..
2025-03-15 13:16:01,695 [INFO] USER:racharla_sanjana base_generator.py:223:_validate_resource_name LOG: Validating resource name: 'lambda_topic'
2025-03-15 13:16:01,695 [INFO] USER:racharla_sanjana base_generator.py:239:_validate_resource_name LOG: Resource name: 'lambda_topic' passed the validation
2025-03-15 13:16:01,696 [INFO] USER:racharla_sanjana base_generator.py:67:_get_deployment_resources_files LOG: Recursively getting all the deployment_resources.json with root 'C:/Users/racharla_sanjana/Documents/Aws/AWS'
2025-03-15 13:16:01,697 [INFO] USER:racharla_sanjana base_generator.py:87:_find_resources_by_type LOG: Looking for resource 'sns_topic' in meta
2025-03-15 13:16:01,704 [INFO] USER:racharla_sanjana base_generator.py:93:_find_resources_by_type LOG: Found '{'C:\\Users\\racharla_sanjana\\Documents\\Aws\\AWS\\task01\\deployment_resources.json': {'lambda_topic'}, 'C:\\Users\\racharla_sanjana\\Documents\\Aws\\AWS\\task01\\app\\lambdas\\hello_world\\deployment_resources.json': set(), 'C:\\Users\\racharla_sanjana\\Documents\\Aws\\AWS\\task02\\deployment_resources.json': set(), 'C:\\Users\\racharla_sanjana\\Documents\\Aws\\AWS\\task02\\app\\lambdas\\hello_world\\deployment_resources.json': set(), 'C:\\Users\\racharla_sanjana\\Documents\\Aws\\AWS\\task03\\deployment_resources.json': set(), 'C:\\Users\\racharla_sanjana\\Documents\\Aws\\AWS\\task03\\app\\lambdas\\hello_world\\deployment_resources.json': set(), 'C:\\Users\\racharla_sanjana\\Documents\\Aws\\AWS\\task04\\deployment_resources.json': {'lambda_topic'}, 'C:\\Users\\racharla_sanjana\\Documents\\Aws\\AWS\\task04\\app\\lambdas\\sns_handler\\deployment_resources.json': set(), 'C:\\Users\\racharla_sanjana\\Documents\\Aws\\AWS\\task04\\app\\lambdas\\sqs_handler\\deployment_resources.json': set()}' inside with type 'sns_topic'
2025-03-15 13:16:01,704 [INFO] USER:racharla_sanjana base_generator.py:103:_get_resource_meta_paths LOG: Looking for sns_topic 'lambda_topic' in meta...
2025-03-15 13:16:01,704 [INFO] USER:racharla_sanjana base_generator.py:106:_get_resource_meta_paths LOG: Found 'lambda_topic' in meta from 'C:\Users\racharla_sanjana\Documents\Aws\AWS\task01\deployment_resources.json'
2025-03-15 13:16:01,704 [INFO] USER:racharla_sanjana base_generator.py:106:_get_resource_meta_paths LOG: Found 'lambda_topic' in meta from 'C:\Users\racharla_sanjana\Documents\Aws\AWS\task04\deployment_resources.json'
2025-03-15 13:16:01,704 [INFO] USER:racharla_sanjana base_generator.py:108:_get_resource_meta_paths LOG: Found sns_topic 'lambda_topic' in these files: '['C:\\Users\\racharla_sanjana\\Documents\\Aws\\AWS\\task01\\deployment_resources.json', 'C:\\Users\\racharla_sanjana\\Documents\\Aws\\AWS\\task04\\deployment_resources.json']'
2025-03-15 13:16:01,704 [WARNING] USER:racharla_sanjana base_generator.py:214:_find_file_with_duplicate LOG: Duplicated sns_topic with name 'lambda_topic' was found in: ['C:\\Users\\racharla_sanjana\\Documents\\Aws\\AWS\\task01\\deployment_resources.json', 'C:\\Users\\racharla_sanjana\\Documents\\Aws\\AWS\\task04\\deployment_resources.json']
2025-03-15 13:16:01,704 [WARNING] USER:racharla_sanjana base_generator.py:189:write LOG: Found duplicate while generating meta. Resource with name 'lambda_topic' was found in file 'C:\Users\racharla_sanjana\Documents\Aws\AWS\task01\deployment_resources.json'.
2025-03-15 13:16:03,821 [WARNING] USER:racharla_sanjana base_generator.py:191:write LOG: Overwriting resource 'lambda_topic'
2025-03-15 13:16:03,822 [INFO] USER:racharla_sanjana base_generator.py:152:generate_deployment_resource LOG: Generating deployment resources for sns_topic 'lambda_topic'
2025-03-15 13:16:03,822 [INFO] USER:racharla_sanjana base_generator.py:160:_generate_resource_configuration LOG: Generating configuration for sns_topic 'lambda_topic'
2025-03-15 13:16:03,822 [INFO] USER:racharla_sanjana base_generator.py:37:_resolve_configuration LOG: Resolving not required params...
2025-03-15 13:16:03,823 [INFO] USER:racharla_sanjana base_generator.py:57:_resolve_configuration LOG: Setting given value 'eu-central-1' for param 'region'
2025-03-15 13:16:03,823 [INFO] USER:racharla_sanjana base_generator.py:44:_resolve_configuration LOG: Setting default value - the object '[]' of the class '<class 'list'>' for param 'event_sources'
2025-03-15 13:16:03,823 [INFO] USER:racharla_sanjana base_generator.py:44:_resolve_configuration LOG: Setting default value - the object '{}' of the class '<class 'dict'>' for param 'tags'
2025-03-15 13:16:03,823 [INFO] USER:racharla_sanjana base_generator.py:202:write LOG: Writing deployment resources for sns_topic 'lambda_topic' to the file 'C:\Users\racharla_sanjana\Documents\Aws\AWS\task01\deployment_resources.json'
2025-03-15 13:16:03,824 [INFO] USER:racharla_sanjana meta.py:489:sns_topic LOG: SNS topic 'lambda_topic' was added successfully
2025-03-15 13:16:03,825 [INFO] USER:racharla_sanjana helper.py:330:timed LOG: Stage sns_topic, elapsed time: 0:00:02.129270
